src/main/java/net/sf/jabref/external/FindFullTextAction.java

```diff
@@ -44,6 +44,20 @@ public void init() throws Throwable {
 
     @Override
     public void run() {
+        if (basePanel.getSelectedEntries().size() >= 10) {
```

Comment:
Please extract that "magic number" to a constant.
https://en.wikipedia.org/wiki/Magic_number_(programming)